common/src/main/java/feast/common/models/Feature.java

```diff
@@ -14,25 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package feast.serving.util;
+package feast.common.models;
 
-import feast.proto.core.FeatureSetProto.FeatureSetSpec;
 import feast.proto.serving.ServingAPIProto.FeatureReference;
 
-public class RefUtil {
-  public static String generateFeatureStringRef(FeatureReference featureReference) {
+public class Feature {
+
+  /**
+   * Accepts FeatureReference object and returns its reference in String
+   * "project/featureset_name:feature_name".
+   *
+   * @param featureReference {@link FeatureReference}
+   * @return String format of FeatureReference
+   */
+  public static String getFeatureStringRef(
+      FeatureReference featureReference, boolean ignoreProject) {
```

Comment:
`ignoreProject` is undocumented. Is there a reason the change is in this PR?

Reply:
Previously in RequestUtil [here](https://github.com/feast-dev/feast/pull/801/files#diff-04c8ea2f153f6312e191d7cbace3d298L95), this returns a string without the project, whereas other use-cases return with project in the string.

Reply:
Ok, this needs to be cleaned up later though. I would rather use overloaded methods here in any case.

Please make sure to update the comments.

Reply:
Ok, will clean this up. I've updated the docs to explain the parameter for now.